packages/mc-scripts/config/create-webpack-config-for-development.js

```diff
@@ -215,9 +214,12 @@ module.exports = ({
           {
             loader: require.resolve('css-loader'),
             options: {
-              modules: true,
+              modules: {
+                mode: 'local',
```

Comment:
This is the new way of configuring it. css-modules is disabled by default now and the modules is a boolean or object.